README.md

```diff
@@ -20,6 +20,19 @@ channel](https://gophers.slack.com/messages/CA7HK8EEP)** on the [Gophers
 Slack](https://gophers.slack.com/). Sign up
 [here](https://invite.slack.golangbridge.org) first if necessary.
 
+### Management: New Releases
+
+A new set of management libraries that follow the [Azure SDK Design Guidelines for Go](https://azure.github.io/azure-sdk/golang_introduction.html) and based on [Azure Core modules](https://github.com/Azure/azure-sdk-for-go/tree/master/sdk/azcore) are now in Public Preview. These new libraries provide a number of core capabilities that are shared amongst all Azure SDKs, including the intuitive Azure Identity library, an HTTP Pipeline with custom policies, error-handling, distributed tracing, and much more. You can find the list of new packages [on this page](https://azure.github.io/azure-sdk/releases/latest/golang.html). 
```

Comment:
It seems that the link `https://azure.github.io/azure-sdk/releases/latest/golang.html` is invalid now therefore this PR fails on link validation. Could you please take a look?

Reply:
yes, this page is still in progress and not available. will need to work on those pages first before we can merge this PR